README.md

```diff
@@ -26,6 +26,7 @@ On Debian or Ubuntu, this should satisfy most/all dependencies:
 
 ### Compile instructions
 
+    ./autogen.sh
```

Comment:
This should probably be qualified with "if building from git", or broken out separately somehow.

Reply:
I added a comment to the line. It now reads:
```
 +    ./autogen.sh  # from Git only; not necessary if building from tarball 
```